streams/src/main/java/org/apache/kafka/streams/kstream/Windowed.java

```diff
@@ -26,26 +26,36 @@
  */
 public class Windowed<T> {
 
-    private T value;
+    private T key;
 
     private Window window;
 
-    public Windowed(T value, Window window) {
-        this.value = value;
+    public Windowed(T key, Window window) {
+        this.key = key;
         this.window = window;
     }
 
-    public T value() {
-        return value;
+    /**
+     * Return the key of the window.
+     *
+     * @return the key of the window
+     */
+    public T key() {
```

Comment:
Should this method be called `get()` instead (similar to the upcoming `Bytes#get()`)? /cc @guozhangwang 

There's nothing key-specific in this implementation.